utils.py

```diff
@@ -365,9 +365,11 @@ def convert_to_hash(input_string, max_length):
             (input_string, type(input_string)))
 
     # Encodes strings using the character set [A-Za-z0-9].
+    # Prefixing altchars with b' to stay compatible with other bytestring
```

Comment:
Isn't this more along the lines of: `Prefixing altchars with b' to ensure that all characters in encoded_string remain encoded (otherwise encoded_string would be of type unicode).`

(The idea is that altchars is used to substitute the last 2 chars in the base64 charset.)

Reply:
Done